feature/src/main/java/com/gdg/feature/map/MapRoute.kt

```diff
@@ -81,21 +124,250 @@ fun MapScreen(
+                style = CrowdZeroTheme.typography.h1JalnanGothic,
+                color = CrowdZeroTheme.colors.white,
+                modifier = Modifier.graphicsLayer(rotationZ = -2f)
+                    .offset(x = 2.dp, y=-27.dp)
+            )
+
+            Text(
+                text = stringResource(R.string.splash_subtitle),
+                style = CrowdZeroTheme.typography.c4Regular,
+                color = CrowdZeroTheme.colors.white,
+                modifier = Modifier.offset(x=20.dp, y=-35.dp)
+
+            )
+
+
+        }
+
+
+        Image(
+            painter = painterResource(id = R.drawable.ic_buildings),
+            contentDescription = stringResource(id = R.string.splash_desc),
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(360.dp)
+                .align(Alignment.BottomCenter)
+                . offset(y = 17.dp)
         )
-        Button(
-            onClick = { onButtonClick(1) },
+    }
+}
+
+@Composable
+fun PlaceInfoCard(place: PlaceEntity?, modifier: Modifier = Modifier,onButtonClick: (Int) -> Unit) {
+    if (place == null) return
+
+    Box(
+        modifier = modifier
+            .width(500.dp)
+            .height(170.dp)
+            .padding(horizontal = 15.dp, vertical = 20.dp)
+            .graphicsLayer {
+                shadowElevation = 20f
+                shape = RoundedCornerShape(20.dp)
+                clip = false
+            }
+            .shadow(
+                elevation = 20.dp,
+                shape = RoundedCornerShape(20.dp),
+                ambientColor = Color.Black.copy(alpha = 1f),
+                spotColor = Color.Black.copy(alpha = 4f)
+            )
+            .background(CrowdZeroTheme.colors.white, shape = RoundedCornerShape(20.dp)),
+        contentAlignment = Alignment.TopStart
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.ic_nextbuildings),
+            contentDescription = "배경 빌딩 아이콘",
             modifier = Modifier
-                .align(Alignment.BottomEnd)
+                .fillMaxSize()
+                .align(Alignment.BottomEnd),
+            contentScale = ContentScale.Crop,
+        )
+
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 20.dp),
+            verticalArrangement = Arrangement.Top
         ) {
-            Text(text = "상세 페이지로 이동")
+
+            Row(
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Spacer(modifier = Modifier.weight(1f))
+                Image(
+                    painter = painterResource(id = R.drawable.ic_right_arrow),
+                    contentDescription = stringResource(R.string.place_info_detail_view),
+                    modifier = Modifier
+                        .size(35.dp)
+                        .padding(start = 15.dp, top = 15.dp)
+                        .noRippleClickable { onButtonClick(1) },
```

Comment:
P1: 1이 아니라 선택된 장소의 id를 다음 화면으로 넘겨줘야겠죠? 그리고 아이콘 클릭해야 다음 화면 넘어가는 것보다 PlaceInfoCard 자체를 선택했을 때 화면 넘어가는게 나아서 클릭 리스너 상위 레이아웃으로 옮겨주세요!!
```suggestion
                        .noRippleClickable { onButtonClick(place.id) },
```